extension/src/helpers/stellar.ts

```diff
@@ -23,6 +23,7 @@ export const getTransactionInfo = (search: string) => {
   const transactionInfo = parsedSearchParam(search);
 
   const {
+    accountToSign,
```

Comment:
for me accountToSign is showing up undefined even when I give one. Is it showing up for you?

Reply:
user error